### ISSUE TITLE
jbrowse: add 'product' to style:{description} list

### ISSUE THREAD

**Issue by tseemann** [galaxyproject/tools-iuc#795]
Microbial genomics use the `/product` field for protein names. Please add it to the default tags to label features please, as below:

```
  "style": {
    "label": "name,id",
    "description": "product,note,description",
}
```

**Comment by erasche** [comment]
And us viral people do as well :) However, when I built the tool, I tried to wrap it fairly closely to the default JBrowse configuration. [Upstream](http://gmod.org/wiki/JBrowse_Configuration_Guide#HTMLFeatures_Configuration_Options) had chosen that as their default and given that it was exposed as a configurable option, I had no strong rationale from deviating from their default. I'll see if I can add this to the next round of updates, but I don't wish to make this tool any more specific to one group than another (w/r/t default configuration).

https://github.com/galaxyproject/tools-iuc/blob/master/tools/jbrowse/macros.xml#L221

**Comment by tseemann** [comment]
My argument would be that `product` is the official INDSC (Genbank etc) field for the protein name, which is what you want shown in a `CDS` feature. 


`description` is not official GFF or anything?

**Comment by erasche** [comment]
> which is what you want shown in a CDS feature. 

Oh, yes, definitely. I come from a ... very limited world view (only viral data), and was trying not to encode any of my prior assumptions about data/field usage. Adding product should be a relatively low impact change.

> description is not official GFF or anything?

Indeed, that does make it an odd default.

**Comment by nsoranzo** [comment]
Should be fixed by #851.
